internal/commands/publish.go

```diff
@@ -374,35 +384,41 @@ func endOfSupportFor(publishDate time.Time) string {
 	return endOfNinthMonth.Format(publishDateFormat)
 }
 
-func (p Publish) attachLicenseFile(slug string, releaseID int, version *releaseVersion) (string, error) {
-	if version.IsGA() {
-		productFiles, err := p.PivnetProductFilesService.List(slug)
-		if err != nil {
-			return "", err
-		}
+func (p Publish) attachLicenseFiles(slug string, releaseID int, version *releaseVersion) ([]string, error) {
+	var attachedLicenseFileNames []string
 
-		productFile, found := findMatchingOSL(productFiles, version)
-		if !found {
-			return "", errors.New("required license file doesn't exist on Pivnet")
-		}
+	if !version.IsGA() {
+		return []string{}, nil
```

Comment:
you can `return nil, nil`. Slices are fancy pointers so this does more than what's needed. It fine to do this but it adds an allocation.